### PR TITLE
Context ordering

### PR DESCRIPTION
**Pull request by skyerus**

### CODE REVIEW

**Comment by skyerus** [comment]
Draft as dependent on https://github.com/open-feature/golang-sdk/pull/42

File: pkg/openfeature/client.go [hunk 1]
```diff
@@ -304,6 +304,10 @@ func mergeContexts(evaluationContexts ...EvaluationContext) EvaluationContext {
 	mergedCtx := evaluationContexts[0]
 
 	for i := 1; i < len(evaluationContexts); i++ {
+		if mergedCtx.TargetingKey == "" && evaluationContexts[i].TargetingKey != "" {
+			mergedCtx.TargetingKey = evaluationContexts[i].TargetingKey
```
Comment (james-milligan) [inline pkg/openfeature/client.go:308]: minor, but could add a break in here

Comment (skyerus) [reply pkg/openfeature/client.go:308]: we want to keep merging the attributes, a break would end that

Comment (james-milligan) [reply pkg/openfeature/client.go:308]: ah!

**Review by james-milligan — APPROVED** [review]

**Pull request merged**